doc/rolling_update.md

```diff
@@ -0,0 +1,279 @@
+# Rolling update
+
+Here is described rolling update for application that uses
+[multiversion approaach](/doc/multiversion.md) or "classic" RPM or DEB packages.
+
+## Requirements
+
+* `tarantool.cartridge` >= `1.9.0`
+
+## Leaders promotion
+
+If stateful failover is enabled then for leaders promotion
```

Comment:
It might be better to say that you can use failover_promote ONLY if the stateboard used

Reply:
Only for stateful failover (stateboard is just one of possible state providers).

Reply:
Sorry, "ONLY if stateful failover is enabled"

Reply:
failover_promote step can be used for leaders promotion only if stateful failover is enabled